src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -0,0 +1,75 @@
+
+You can also jump into this experience directly from errors inbox by clicking the **Open in IDE** button next to any stack trace. From there you can ask New Relic AI follow-up questions, or bring teammates into the discussion.
+
+## Sample questions [#sample-questions] 
+
+If you want to get to the root cause of latency issues for your app, here are some good questions to ask:
+
+* What are some recent incidents in my system?
+* Which infrastructure hosts are showing signs of degradation over the past hour?
+
+Our AI chat experience can also help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
+
+* How can I use Terraform to create dashboards? 
+* How can I improve my alerts setup?
+* How can I instrument AWS?
+
+Our AI can also be your partner in observability maturity. If you're responsible for making sure that your app can handle spikes for a big e-commerce event like Black Friday, it's important to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
+
+* How can I improve the performance of my infrastructure?
+* Were unique user sessions up or down last week compared to the week before?
+
+## Secure data
+
+We understand that data security and privacy are primary concerns for anyone building and maintaining software. We want you to know that anything you ask our AI is safe. NRAI is built with the same security controls and standards that safeguard New Relic customer data. This means that we will never share your data with other accounts or customers.  We work with vendors that meet our privacy and security standards, and we carefully assess prospective vendors in these critical areas before engaging their services. 
+
+Our AI uses a variety of inputs—like this documentation, a large library of example queries, and your telemetry data—to improve it's responses and always provide the best answer. But even though our AI scans your data as needed to provide context and help you build charts and understand response times, that data is _never stored_ by New Relic.  
+
+## Always learning 
+
+New Relic AI is becoming more accurate and responsive the more it's used. We want to build the AI experience you want. When chatting with our AI, please give its response a thumbs up or a thumbs down.  If you notice something that isn't right or that could be improved, let us know. 
+
+Also, New Relic AI has the capacity to recognize multiple languages and respond in them. you have use cases for multi-lingual support, please test them with our AI and give us feedback 
+
+* Go to the user menu, click **Help**, then click **Give us feedback**.
```

Comment:
I'd move this to the end of the first paragraph of this section since I wanted to know how to do this there. You don't need to make it a bulleted item.